sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/appplatform/samples/ManageSpringCloud.java

```diff
@@ -269,7 +269,6 @@ public static void main(String[] args) {
 
             final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
             final TokenCredential credential = new DefaultAzureCredentialBuilder()
-                .authorityHost(profile.environment().getActiveDirectoryEndpoint())
```

Comment:
after azure.identity 1.10.0, we'll need get this line of code back right?

Reply:
Yes, it could be add back.